packages/vats/src/localchain.js

```diff
@@ -63,9 +66,22 @@ const prepareLocalChainAccount = zone =>
         const bankManager = getPower(powers, 'bankManager');
 
         const allegedBrand = await E(payment).getAllegedBrand();
-        const bankAcct = E(bankManager).getBankForAddress(address);
-        const allegedPurse = E(bankAcct).getPurse(allegedBrand);
-        return E(allegedPurse).deposit(payment);
+        const acctsBank = E(bankManager).getBankForAddress(address);
+        const purse = E(acctsBank).getPurse(allegedBrand);
+        return E(purse).deposit(payment);
+      },
+      /**
+       * Withdraw a payment from the account's bank purse.
+       *
+       * @param {Amount} amount
+       */
+      async withdraw(amount) {
+        const { address, powers } = this.state;
+        const bankManager = getPower(powers, 'bankManager');
+
+        const acctsBank = E(bankManager).getBankForAddress(address);
```

Comment:
This `getBankForAddress()` call should happen before this account object is created. No account should have access to everybody else's purses. **critical**

(Didn't I give this feedback a while ago? hm.)